### DISCUSSION TITLE
lfs not building

### DISCUSSION THREAD

**Discussion by gweatherson** [vercel/community#3949] [Help]
### Summary

Astro site, it was deploying fine.

Now it's stuck on Building:

`Cloning github.com/gweatherson/glennscrapastro (Branch: main, Commit: a072913)`
`Skipping build cache, deployment was triggered without cache.`
`Error: Cloning failed`

The difference is lfs.
Enabled lfs on the repo: *.jpg filter=lfs diff=lfs merge=lfs -text
Enabled lfs in Vercel

### Example

https://github.com/gweatherson/glennscrapastro

### Steps to Reproduce

Run:

git lfs track "*.jpg"
enable lfs in vercel settings
push changes

**Comment by gweatherson** [comment]
weird, using the vercel cli works.

**Comment by amyegan** [comment]
Hey @gweatherson. Can you double check that [Git Large File Storage (LFS)](https://vercel.com/docs/projects/overview#git-large-file-storage-lfs) is enabled in project settings?

**Reply by gweatherson** [reply]
Thanks @amyegan, it is:
<img width="975" alt="Screenshot 2023-09-06 at 8 45 01 AM" src="https://github.com/vercel/community/assets/1322939/85c1d302-f875-4283-bcf3-3966c49d4a63">
